backend/resources/views/home.blade.php

```diff
@@ -5,11 +5,27 @@
 @parent
 @endsection
 @section('content')
-@include('components.forMembers.pageTitle', ['title'=>'U-laniwa ホーム'])
+@include('components.forMembers.pageTitle', ['title'=>'ホーム'])
+
 
 <div class='w-96'></div>
 
 <div class="belongsProject">
+    {{--@include('components.forMembers.userTimeline')--}}
+    <div class="userTimeline bg-bg-sub w-full p-8 rounded-3xl mb-20">
+        <h2 class="text-xl">タイムライン</h2>
+        <div class="tree">
+
+            @foreach ($timelines as $timeline)
+            @include('components.forMembers.timelineItem',[
+            'date'=>$timeline->start_date,
+            'name'=>$userInfo->last_name,
```

Comment:
$userInfoだとログイン中のユーザーの情報になってしまうので
```suggestion
            'name'=>$userInfo->last_name,
```
こちらでお願いします！

Reply:
　`$user`ですかね…？

Reply:
コピーミス！！ごめん！！

```
      'name'=>$timeline->user->name
```
です！！